src/3d/config_scope_3d.h

```diff
@@ -163,6 +164,17 @@ class DebugDraw3DScopeConfig : public RefCounted {
 	Ref<DebugDraw3DScopeConfig> set_text_outline_size(int32_t _value) const;
 	int32_t get_text_outline_size() const;
 
+	/**
+	 * Makes the text in DebugDraw3D.draw_text the same size regardless of distance.
+	 *
+	 * ![](docs/images/classes/DrawTextFixedSize.webp)
```

Comment:
I can generate this animation myself later, but in this case, please indicate next to or instead of the path `TODO:`.